### DISCUSSION TITLE
Homage to the Square: Exercise 1

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#2] [Show and tell]
1. Open the file `exercises/02-homage-to-the-square/exercise-1.Rmd`
2. Update the function `make_my_homage()` to take a `seed` argument, and set the seed in the body of the function
3. Verify that running `make_my_homage()` with the same `seed` creates the same output, and that with a different seeds creates different outputs
4. Share an output and the seed that created it in this discussion post

**Comment by Kim-s-h** [comment]
![ex1](https://github.com/posit-conf-2023/creative-coding/assets/22897443/9a0adc4c-d9fb-4498-beef-8e48d0839ce8)

seed=1432

**Comment by bshk93** [comment]
![homage_square_2](https://github.com/posit-conf-2023/creative-coding/assets/65432465/84ae551c-11e2-495b-9f2f-597c43f43923)

Seed: 10

**Comment by natalieoshea** [comment]
![ex1-homage-8675309](https://github.com/posit-conf-2023/creative-coding/assets/10425075/4997f6bd-5d61-43a8-8e69-c25ec0416b15)

Seed: 8675309

**Comment by maxrogow** [comment]
seed: 3
![my_homage_square](https://github.com/posit-conf-2023/creative-coding/assets/145362959/c916f0ef-4fef-4fd3-be71-20bc012d116e)

**Comment by dougtommet** [comment]
![square_homage](https://github.com/posit-conf-2023/creative-coding/assets/7944225/fc1d3c49-4e12-4e76-8e54-5d46b90e634c)
seed: 2469

**Comment by thomaspara** [comment]
I did seed = 1999
![ex1-homage-1999](https://github.com/posit-conf-2023/creative-coding/assets/10082327/30bfd1c0-270e-4631-bbd9-c6901c0ac401)

**Comment by lwdozal** [comment]
Seed - 47
![homage47](https://github.com/posit-conf-2023/creative-coding/assets/19821171/76173293-3236-44d2-a748-6ec9344d3c9e)

**Comment by datatoolbox** [comment]
![000016](https://github.com/posit-conf-2023/creative-coding/assets/5366795/45708d94-abeb-4325-aea6-51ff81ed2254)

**Comment by roBertus31** [comment]
![ex1-homage-102285](https://github.com/posit-conf-2023/creative-coding/assets/25853405/9385b74d-59d8-4bad-a5ab-cbd7c018e3fe)

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/393db152-12fc-4804-baea-4deb74f4105c)

**Comment by sshaughnessy** [comment]
![seed-99](https://github.com/posit-conf-2023/creative-coding/assets/36434962/4c1dd1a1-33b3-4810-b75b-e19453ea95f9)

seed 99

**Comment by MHanken** [comment]
seed = 5683

![Rplot](https://github.com/posit-conf-2023/creative-coding/assets/49319823/7750932b-9334-4df0-b4ab-5dfd0a2197d3)

**Reply by MHanken** [reply]
After explicitly passing 5683 as the seed argument number...
![homage](https://github.com/posit-conf-2023/creative-coding/assets/49319823/42b24d91-2d05-41f0-b451-25d97856732e)

**Comment by Meghansaha** [comment]
Seed = 9999 

![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/c44a1a00-dda9-4141-a385-dc1f0972d8c3)

**Comment by datatoolbox** [comment]
![ex2-homage-12](https://github.com/posit-conf-2023/creative-coding/assets/5366795/67bf5ea0-a129-4b55-91f1-21e5d6772d6f)
